ui/components/connections/index.js

```diff
@@ -600,6 +684,19 @@ function Connections({ classes, updateProgress, /*onOpenCreateConnectionModal,*/
     }
   };
 
+  const handleKubernetesClick = (name, connectionID) => {
```

Comment:
Why there is need to redefine if you can use from ConnectionWizard or use `useKubernetes` hook (inside ui/hooks dir) (this will be ideal).

Reply:
Switched to the new k8s hook 👍